examples/exemplars/main.go

```diff
@@ -34,6 +34,14 @@ func main() {
 		Help:    "A histogram of the HTTP request durations in seconds.",
 		Buckets: prometheus.ExponentialBuckets(0.1, 1.5, 5),
 	})
+	requestsTotal := prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
I'm not sure I understand how the changes in this file relate to the rest of the PR.

Reply:
I would recommend writing a unit tests(s) that will verify those instead. Manual checks won't get us far, ideally we want next commits to be also validated against potential issues with created timestamps.

Reply:
tests added :)